### DISCUSSION TITLE
Dashboards?

### DISCUSSION THREAD

**Discussion by elbamos** [quarto-dev/quarto-cli#2158] [Q&A] [answered]
The FAQ for rmarkdown users says that dashboarding capability will arrive in Fall, 2022, which is right around the corner. Is there a branch or a beta or an alpha that we can play with and provide feedback on?

**Comment by mcanouil** [comment]
The FAQ is not exactly up to date.
There is a trade off between fixing all the issues unveiled since Quarto 1.0 release last month and adding new features/bugs.

They will announce in due date, also you can watch this repository and see what's in the current pre-release.

**Comment by jjallaire** [comment, marked as answer]
Quarto Dashboards are available in the v1.4 pre-release: https://quarto.org/docs/dashboards/
